electron/SatelliteApp.ts

```diff
@@ -615,9 +615,16 @@ export class SatelliteApp {
         this.airflowSettings = this.store.get('airflowSettings');
 
         this.store.set('latestUpdateVersion', appVersion);
+        
+        // Need to make sure that clean_dag_run.py will be updated (init creates new if not present)
+        try {
+            await fs.promises.unlink(`${this.airflowSettings.AIRFLOW_HOME}/dags/clean_dag_run.py`)
+        } catch (e) {
+            Log.info('Failed to remove clean_dag_run.py');
+        }
 
         const init_commands = [
-            `cwl-airflow init`
```

Comment:
Do we need to replace init with init --update ? It's the first start no anything

Reply:
If there is no any DAGs in dags folder `--upgrade` won't do anything, so you can leave just `cwl-airflow init`